### ISSUE TITLE
Umlauts issue

### ISSUE THREAD

**Issue by emmamarichal** [alexeiva/philosopher#6]
Hi @alexeiva! I hope you're well.

Several users mentioned some issues with the Umlauts:
- https://github.com/google/fonts/issues/1225
- https://github.com/google/fonts/issues/7466

Do you think this is something that can be fixed quickly? If you don't have time to do it, I can fix it directly on your repo and make a small update.
I could also take the opportunity to take a look to the other issue opened in the repo.

Let me know, 
Cheers!

**Comment by alexeiva** [comment]
Yes, I'm able to fix this quickly, will check the other issues as well.

**Comment by emmamarichal** [comment]
@alexeiva excellent, thank you :)

**Comment by alexeiva** [comment]
<img width="406" alt="Screenshot 2024-06-07 at 12 16 26" src="https://github.com/alexeiva/philosopher/assets/16634089/da44cb81-61e0-4fe1-b808-af6fa5ecaf22">

fixed in sources

**Comment by emmamarichal** [comment]
excellent, let me export the fonts and do a quick QA!
